src/Sylius/Bundle/CoreBundle/Resources/config/validation/AdminUser.xml

```diff
@@ -51,5 +51,8 @@
                 <option name="groups">sylius</option>
             </constraint>
         </property>
+        <property name="timezone">
+            <constraint name="Timezone"/>
```

Comment:
I believe we also need an API Behat scenario to check this behaviour. I think it would not work right now 😄 as it does not have a proper `sylius` validation group defined